index.html

```diff
@@ -124,8 +124,9 @@ <h5>About</h5>
       vuetify: new Vuetify(),
       data: () => {
         let rawItems = [
+          ['./img/journey/journey.jpg', 'The Journey', 'Guide Dogs of America', 'https://www.guidedogsofamerica.org/', 'Mar 26, 2023'],

```

Comment:
The new poster has a different ratio, so I had to stretch the dawn's a bit to make them the same, since they show up side by side on home.